### PR TITLE
Create Registration (Response) Controller #285

### PR DESCRIPTION
**Pull request by ilzeLourens**

### CODE REVIEW

File: api/app/registrationResponse/api.py [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+            # is the relevant answer
+            summary = ""
+            for answer in answers:
+                for question in questions:
+                    if answer.registration_question_id == question.id:
+                        summary += "Question heading :" + question.headline + "\nQuestion Description :" + \
+                           question.description + "\nAnswer :" + _get_answer_value(
+                                answer, question) + "\n"
+
+            subject = 'Registration'
```
Comment (jaderabbit) [inline api/app/registrationResponse/api.py]: Maybe update to "<Event Name> Registration>"?

**Comment by jaderabbit** [comment]
Just a few things!
You write very clean code :clap: :smiley:

File: api/app/registrationResponse/api.py [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+            registration_form = db.session.query(RegistrationForm).filter(RegistrationForm.id == registration.
+                                                                          registration_form_id).first()
+
+            if registration_form is None:
+                return errors.REGISTRATION_FORM_NOT_FOUND
+            db_answers = db.session.query(RegistrationAnswer).filter(RegistrationAnswer.registration_id ==
+                                                                     registration.id).all()
+
+            response = {
+                'registration_id': registration.id,
```
Comment (avishkar58) [inline api/app/registrationResponse/api.py]: Please also add "Registration.confirmed" and the created_at datetime to the response

File: api/app/registrationResponse/api.py [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+
+                    answer = RegistrationAnswer(registration_id=registration.id,
+                                                registration_question_id=answer_args['registration_question_id'],
+                                                value=answer_args['value'])
+
+                    db.session.add(answer)
+            db.session.commit()
+            return 200
+        except Exception as e:
+            return 'Could not access DB', 400
```
Comment (avishkar58) [inline api/app/registrationResponse/api.py]: Please log the exception message

**Pull request merged**